addons/mail/static/src/components/message/message.scss

```diff
@@ -77,6 +78,7 @@
 .o_Message_headerCommands {
     margin-inline-end: map-get($spacers, 2);
     align-self: center;
+    border: none !important;
```

Comment:
important is a keyword we should never use, if this has to override something, it's better to "remove the something" instead or add it only conditionally